src/screens/Booking/components/PassengerForm/Form.styles.ts

```diff
@@ -0,0 +1,51 @@
+import {StyleSheet, ViewStyle} from 'react-native';
+import {colors, textStyles, responsive} from '@constants';
+
+const innerMargin = responsive.isSmallScreen ? 0 : 24;
+
+const buttonCommonStyles: ViewStyle = {
```

Comment:
why it should be reusable? If it's reusable style variable - please consider to keep that in separate file